locale/en/blog/release/v5.5.0.md

```diff
@@ -15,7 +15,7 @@ author: Evan Lucas
 - **http**: improves expect header handling (Daniel Sellers) [#4501](https://github.com/nodejs/node/pull/4501)
 - **node**: allow preload modules with -i (Evan Lucas) [#4696](https://github.com/nodejs/node/pull/4696)
 - **v8,src**: expose statistics about heap spaces (`v8.getHeapSpaceStatistics()`) (Ben Ripkens) [#4463](https://github.com/nodejs/node/pull/4463)
-* Minor performance improvements:
+- Minor performance improvements:
```

Comment:
This probably shouldn't be in this PR and tackled in a separate patch altogether.

Reply:
It needs to be, otherwise this is considered a separate list and needs spacing

Reply:
Weird, sounds like a bug in markdownlint but fair enough.

Reply:
no, changing list styles creates new lists in markdown

Reply:
Didn't know that, I guess you do learn something new every day :)

Reply:
Oh, I think i was seeing this as a different case from the short diff. It's only really an issue switching -/* in the same list

Reply:
This fails now. https://travis-ci.com/nodejs/nodejs.org/builds/128525665#L283